docs/DeveloperGuide.adoc

```diff
@@ -697,20 +698,21 @@ Use case ends.
 . Coupon Stash works on `_common operating systems
 (OS)_` that have `_Java 11_` or above installed.
 
-. Coupon Stash can hold at least 500 coupons comfortably, and still operate without noticeable lag when entering commands or interacting with the UI.
+. Coupon Stash can store at least 500 coupons without crashing the application.
+. Coupon Stash can operate without noticeable lag (~2s) when entering commands or interacting with the UI.
 . Coupon Stash caters to users who have above average typing speed, and these users should be able to get tasks completed faster in the application by typing, rather than using the mouse and the UI.
 . Coupon Stash source code should be covered by tests as much as possible.
-. Stash should be easy to use for users, who are not familiar with coding.
+. Coupon Stash should be easy to use for users, who are not familiar with coding.
 . All monetary amounts should be accurate up to 2 decimal places.
 . Coupon Stash should be portable.
-. Data files should remain unchanged when moving from OS to OS.
+. Data files should remain unchanged when transferring from OS to OS.
 . Coupon Stash works perfectly without access to the internet.
 . Coupon Stash supports various types of coupons (e.g. promotional codes, QR code, or barcode) (coming in in v2.0)
 
 [appendix]
 == Glossary
 
-Terms that are italicised inline code (e.g `_operating system_`) can be found with definitions here.
+[red]# Terms that are italicised inline code (e.g `_operating system_`) can be found with definitions here. #
```

Comment:
This doesn't not appear as red colour text leh, but instead the [red]# is shown on the website

Reply:
@kevvvinn please resolve this!